wdl/lift.wdl

```diff
@@ -230,8 +230,8 @@ task lift_postprocess {
         pos_col = "~{pos_col}"
         ref_col = "~{ref_col}"
         alt_col = "~{alt_col}"
-        af_col = "~{af_col}"
-        beta_col = "~{beta_col}"
+        af_col = ~{if defined(af_col) then "'af_col'" else "None"}
```

Comment:
this is not setting to hardcoded 'af_col' in case the variable af_col is defined. 
Needs to be ~{if defined(af_col) then "'${af_col}'" else "None"}

Reply:
Yes